extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -443,6 +443,7 @@ describes.realWin('amp-ad-exit', {
   });
 
   it('should replace custom URL variables with vars', () => {
+    toggleExperiment(win, 'url-replacement-v2', true);
```

Comment:
should we add this to beforeEach so that we turn it on for all test cases?

Reply:
Yes we probably should, done.